diff --git a/CMakeLists.txt b/CMakeLists.txt
index 70f51c6..38b2a07 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,6 @@ ENDIF()
 ### LIBRARY #################################################################

 # definitions to pass to the compiler
-ADD_DEFINITIONS( "-Wall -ansi -pedantic -std=gnu++11" )
 ADD_DEFINITIONS( "-Wno-long-long -Wno-effc++ -Wno-unused-parameter" )
 if( ZSEGMENTED )
     message ( STATUS "Enabled sensor segmentation" )
